### ISSUE TITLE
Generate report file for diffs only

### ISSUE THREAD

**Issue by bitvale** [tomasbjerre/violations-gradle-plugin#18]
Is it possible to generate `codeClimateFile` only for diffs? Now it generates a file with all violations, but in log shows only diff.

**Comment by tomasbjerre** [comment]
No, now it will always put all violations in the `codeClimateFile`.

But it can be implemented. Perhaps best to add a `diffCodeClimateFile` config.
